include/flatbuffers/flatbuffers.h

```diff
@@ -1294,9 +1300,30 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     return true;
   }
 
+  // Returns the message size in bytes
+  size_t GetSize() const {
```

Comment:
Done, I also renamed GetBuffer -> GetInternalBuffer and GetBufferSize -> GetInternalBufferSize